addons/calendar/views/mail_activity_views.xml

```diff
@@ -34,4 +34,22 @@
         </field>
     </record>
 
+    <record id="mail_activity_view_form" model="ir.ui.view">
+        <field name="name">mail.activity.view.form.inherit.calendar</field>
+        <field name="model">mail.activity</field>
+        <field name="inherit_id" ref="mail.mail_activity_view_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//button[@name='action_done']" position="attributes">
+                  <attribute name="attrs">{'invisible': ['|', ('activity_category', '=', 'meeting'), ('force_next', '=', True)]}</attribute>
+            </xpath>
+            <xpath expr="//button[@name='action_done']" position="before">
+                <button string="Open Calendar"
+                    attrs="{'invisible': [('activity_category', '!=', 'meeting')]}"
+                    name="action_create_calendar_event"
+                    type="object"
+                    class="btn-primary"/>
+            </xpath>
+        </field>
+    </record>
```

Comment:
Hello, could you please explain this change? There was nothing in the spec that gave info about calendar specific change and I don't know this module very well.

Reply:
Hello. We made a new view(mail_activity_view_form) for technical view of activities, before that(old behaviour) when "activity_category" is 'meeting' the mark as done button was invisibled and "Open calendar" button is added to footer in technical view so need to add this inherited view to maintain old behaviour.

Reply:
Shouldn't you simply hide all buttons ? This seems quite complex as I thought purpose was simply to have a form view without buttons.

Reply:
This is exactly the feedback I posted on the task, you might want to check it.